cityofaustin/vision-zero
# Make VZE compatible with the two-layer model

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/19366

## Testing

**URL to test:** 
Local

**Steps to test:**
1. Check out this branch. Spin up recent copy of the prod database. Apply the hasura migrations and metadata.
2. Open the VZE. Check that all pages work, especially the crashes and location list pages, crash details page.
3. Test sorting and filtering on the crashes list page.
4. On the crashes details page test editing the coordinates, the Geocode Provider above the map should update to Manual Geocode whenever you edit coordinates that were previously provided by CRIS
5. Also test updating values in the related records table, data tables, etc.
6. Test swapping the primary and secondary addresses - the [bug](https://github.com/cityofaustin/vision-zero/pull/1551#issue-2525714232) that lead us to implementing the two layer data model is fixed now!! :partying_face: 


---

#### Ship list
